reader_test.go

```diff
@@ -101,9 +101,9 @@ Bob,bob@example.com,25
 
 func TestComplexTypes(t *testing.T) {
 	// CSV data with various types
-	csvData := `name,created_at,active,score,count,rate,tags
-John,2023-01-02T15:04:05Z,true,98.6,42,3.14,test;debug
-Jane,2023-06-15T09:30:00Z,false,75.2,100,2.718,prod;live`
+	csvData := `name,created_at,active,score,count,rate,tags,rank
+John,2023-01-02T15:04:05Z,true,98.6,42,3.14,test;debug,1
+Jane,2023-06-15T09:30:00Z,false,75.2,100,2.718,prod;live,42`
```

Comment:
Could you add a test about sending a negative number via a CSV into a uint field ?

I feel like it could be useful to validate how the library behaves in such use case.

Also, it would help to avoid regression

Reply:
Thanks for the comment. I've added a test for invalid values.